doc/auth.md

```diff
@@ -0,0 +1,71 @@
+# Enable Authentication
+
+Client can communicate with Pravega in a more secure way by enabling Authentication.
+Authentication Handler can be implemented in different ways. For details see [Pravega AUthentication](https://github.com/pravega/pravega/blob/master/documentation/src/docs/auth/auth-plugin.md) for details.
+
+PasswordAuthHandler(default) is the most basic authentication handler.
+For using this, user needs to do the following:
+
+1. Create a userPassword file containing `<user>:<password>:<acl>;` on each line for one user.
```

Comment:
Does "on each line for one user" look OK? a line-per-user looks better to me.

Reply:
Done. Changed to `with one line per user`